contrib/ci/debian_s390x.sh

```diff
@@ -12,6 +12,7 @@ meson .. \
 	--werror \
 	-Dplugin_uefi=false \
 	-Dplugin_dell=false \
+	-Dplugin_redfish=false \
```

Comment:
Are you disabling redfish in here just because it will never physically be on this architecture?  I think there is actually value in running here if possible to prevent endianness problems in the code in case the code ever does need to run on a new architecture in the future.

Reply:
True. I've enabled it now, lets see what explodes.

Reply:
Oh looking at the failure it's because you use efivar now, and that's x86 only stuff.  I guess this idea won't work then (same reason we couldn't do UEFI on this arch), oh well.
`arch: amd64 arm64 armel armhf i386 ia64 x32`

Reply:
We could use the fu-uefi-vars functionality...

Reply:
I guess your call if you think that makes sense.